rca/static_src/sass/components/_card.scss

```diff
@@ -459,6 +459,119 @@
         }
     }
 
+    .index-module & {
+        #{$root}__container {
+            position: relative;
+            margin-bottom: ($gutter * 2);
+
+            @include media-query(large) {
+                flex-direction: row;
+                margin-bottom: ($gutter * 3.5);
+            }
+        }
+
+        #{$root}__content {
+            position: relative;
+            top: -$notch-height-small;
+            margin-bottom: -$notch-height-small;
+
+            @include media-query(medium) {
+                top: -$notch-height-small;
+                margin-bottom: -$notch-height-small;
+            }
+
+            @include media-query(large) {
+                position: static;
+                top: 0;
+                margin-bottom: 0;
+            }
+
+            // One off custom breakpoint between medium and large
+            @media only screen and (max-width: 1180px) and (min-width: 1023px) {
+                padding-top: 0;
+            }
+
+            &::after {
+                @include z-index(base);
+                position: absolute;
+                top: 0;
+                left: 0;
+                content: '';
+                display: block;
+                height: $notch-height-small;
+                width: 50%;
+
+                @include media-query(medium) {
+                    height: $notch-height-medium;
+                }
+
+                @include media-query(large) {
+                    content: none;
```

Comment:
I never thought of doing this to hide the content, always used `display: none`

Reply:
I'm never 100% sure which is best, content: none always feels less heavy to me, but can't say that has any basis!